atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/SweepStatsKeyValueService.java

```diff
@@ -213,16 +242,21 @@ private Runnable createFlushTask() {
         };
     }
 
+    private boolean shouldFlush() {
+        return totalModifications.get() >= WRITE_THRESHOLD || totalModificationsSize.get() >= WRITE_SIZE_THRESHOLD;
+    }
+
     private void flushWrites(Multiset<TableReference> writes, Set<TableReference> clears) {
         if (writes.isEmpty() && clears.isEmpty()) {
-            log.debug("No writes to flush");
+            log.info("No writes to flush");
             return;
         }
 
-        log.debug("Flushing stats for {} writes and {} clears",
-                writes.size(), clears.size());
-        log.trace("Flushing writes: {}", writes);
-        log.trace("Flushing clears: {}", clears);
+        log.info("Flushing stats for {} writes and {} clears",
```

Comment:
Should we continue to log this at debug? It doesn't seem it's going to be very noisy, but it doesn't provide a lot of information by itself.